Atomic/syscontainers.py

```diff
@@ -400,7 +403,10 @@ def _do_checkout(self, repo, name, img, upgrade, values, destination, unitfileou
         manifest_file = os.path.join(exports, "manifest.json")
         if os.path.exists(manifest_file):
             with open(manifest_file, "r") as f:
-                manifest = json.loads(f.read())
+                try:
+                    manifest = json.loads(f.read())
+                except ValueError:
+                    raise ValueError("Invalid manifest.json file in image.")
```

Comment:
Same as above, this would be in the rootfs/exports (also not created since the install fails)